commands.go

```diff
@@ -3349,6 +3352,38 @@ func (c cmdable) FunctionRestore(ctx context.Context, libDump string) *StringCmd
 	return cmd
 }
 
+func (c cmdable) FunctionStats(ctx context.Context) *FunctionStatsCmd {
+	cmd := NewFunctionStatsCmd(ctx, "function", "stats")
+	_ = c(ctx, cmd)
+	return cmd
+}
+
+func (c cmdable) FCall(ctx context.Context, function string, keys []string, args ...interface{}) *Cmd {
+	cmdArgs := fcallArgs("fcall", function, keys, args...)
+	cmd := NewCmd(ctx, cmdArgs...)
+	cmd.SetFirstKeyPos(3)
```

Comment:
should:
```go
// exp: fcall func 0 arg1 arg2
if len(keys) > 0 {
        cmd.SetFirstKeyPos(3)
}
```

Reply:
Yes, we have discovered an issue with eval/eval_ro. When key is not passed, we incorrectly treat arg parameter as key, which is incorrect. We should fix eval/eval_ro.

Reply:
I'll fix eval/eval_ro.